src/agent/rustjail/src/container.rs

```diff
@@ -552,7 +552,7 @@ fn do_init_child(cwfd: RawFd) -> Result<()> {
 
     if guser.additional_gids.len() > 0 {
         setgroups(guser.additional_gids.as_slice()).map_err(|e| {
-            write_sync(
+            let _ = write_sync(
```

Comment:
AFAICT this one is in a function which returns a `Result<>`, so we should be able to report the error.  Is there a reason we don't want to?